### ISSUE TITLE
Get filled color pixel counts

### ISSUE THREAD

**Issue by unitycoder** [unitycoder/UnityMobilePaint#49]
> so my question is :
> If i filled more than one color, then how can i check which color are
> most fill on object? i'm using "Override Test UI".

**easy:**
manually iterate all pixels in the main array and count used pixel colors,
you can do that in another thread.
main pixels array is the: private byte[] pixels; in MobilePaint.cs

**complex:**
check how scene_MobilePaint_LockingMaskWithAreaCalculation.scene does fill amount checking,
using PaintAreaEventListener.cs, then create or modify fill method to count area pixels also,
and could use dictionary to keep track of each color fill counts.
(but if user fills on top of another already filled color, that old color count is not updated..)

**Comment by gohilchirag6565** [comment]
https://www.youtube.com/watch?v=ROzHxAiLXOU
i attach example here,
can you please show with example?

**Comment by unitycoder** [comment]
yes, should use the "easy" method, would be complicated to keep track of pixels otherwise. (and this wouldnt require modifying the plugin source code, can use your own external script to do these)

- have regular c# thread to do the work
- iterate over pixels[] array (or from screencapture texture or rendertexture)
- since you only have 3 possible colors, calculate count for each
